### ISSUE TITLE
Changing entities in your eduGAIN feed

### ISSUE THREAD

**Issue by bradleybeddoes** [ausaccessfed/saml-service#230]
Subject: Changing entities in your eduGAIN feed

Hi,

   we are looking at possibilities of making the aggregation/delivery
process more efficient. Already for a long time we have been using a
simple feature of storing fingerprints calculated on the entity metadata
xml element. We are doing it without any prior normalisation and in most
cases it works just fine. However when we started looking at things more
closely we observed that some AU entities seem to be refreshed very
often. We have observed that on example cases of

entityid = 'https://shibboleth.aarnet.edu.au/idp/shibboleth'
entityid = 'https://idp.flinders.edu.au/idp/shibboleth'

the order of certificates in the entity keeps changing, which of course
changes the fingerprint. There are more of these changing entties, so
these are just examples.

I wonder if this is something that could fixed on your side? It would be
helpful.

Yours

Tomasz Wolniewicz

eduGAIN operations

**Comment by vladimir-mencl-eresearch** [comment]
Hi @bradleybeddoes ,

This should get fixed by the combination of #203 in SAML Service and ausaccessfed/federationregistry#257 in FR.  Not sure if you ever got to update your deployed FR to include that fix - that might be the only thing missing...

Cheers,
Vlad

**Comment by vladimir-mencl-eresearch** [comment]
Hi @rianniello ,

Happy New Year!

When cleaning my inbox, I found a link to this issue ... and I think you might now call it resolved.

The pending part was releasing the changes done to FR in develop (  ausaccessfed/federationregistry#257 ) - and from what I can see externally, this got released in November (I assume after our email exchange).

I see your FR reports version 2.9.1 and I assume this build includes the above fix.

Cheers,
Vlad
